website/blog/2022-06-21-version-069.md

```diff
@@ -0,0 +1,87 @@
+
+<!--truncate-->
+
+## React 18
+
+We are excited to share with you that React Native 0.69 is the first release to support React 18. React 18 has brought [lots of improvements](https://reactjs.org/blog/2022/03/29/react-v18.html), like new hooks such as `useId`. Additionally, React 18 includes new concurrency features such as `useTransition` or full Suspense support.
+
+On React Native 0.69, React 18 is enabled by default. However, if you have not migrated to the New Architecture, you will only be able to leverage the features that don't use concurrent rendering and concurrent features. New Architecture has been built with concurrent rendering in mind but we cannot add that support for the old architecture.
+
+You can learn more about the React 18 support in React Native [here](/docs/next/react-18-and-react-native).
+
+## Bundled Hermes
+
+Before this release, Hermes and React Native were released separately. That led to confusion on which version of Hermes is compatible with which version of React Native. To remedy this issue, starting with React Native 0.69 we will be shipping a compatible version of Hermes alongside React Native. Making this change will make using Hermes in React Native much more stable.
+
+Using the proper version of Hermes is handled by React Native, however, make sure to follow the steps in the [upgrade helper](https://react-native-community.github.io/upgrade-helper/?from=0.68.2&to=0.69.0) to ensure the integration works as intended. If you don't have Hermes enabled already, you can follow the steps [here](/docs/hermes) to do so. While we will continue supporting other JavaScript engines, we recommend everyone to migrate to Hermes to have the best experience and to make sure we can support you better.
+
+Note that users on the New Architecture on Android will need to **build Hermes from source**. For building Hermes from source, Windows users will need to additionally follow [these steps](/architecture/bundled-hermes#android-users-on-new-architecture-building-on-windows).
+
+If you are interested to learn more about how React Native bundles Hermes works under the hood, you can check out the deep-dive documentation [here](/architecture/bundled-hermes).
+
+## New Architecture
+
+We are continuing the roll-out of the New Architecture for both Android and iOS. If you have not migrated your app or library, yet, follow the steps [here](/docs/new-architecture-intro). You can also read the [latest update](/blog/2022/06/16/resources-migrating-your-react-native-library-to-the-new-architecture) on tools and resources for the New Architecture to learn more.
+
+## Highlights of 0.69
+
+As mentioned above, the most important improvements in this release are centered around React 18 support and bundled Hermes. However, there have been other notable changes, including:
+
+- [Deprecating support](https://github.com/facebook/react-native/commit/5f2835b14d35681c268dd64d6ec284ea5f053be3) for iOS/tvOS SDK 11.0, 12.4+
```

Comment:
It isn't clear that we deprecated this version range. This makes it seem like everything newer than 12.4 is deprecated too